Makefile

```diff
@@ -64,7 +64,7 @@ goplugin-example:
 	$(call build,goplugin-example,cmd/goplugin-example/main.go)
 
 PHONY+= lyra
-lyra:
+lyra: check-mods
```

Comment:
Given that `check-mods` is now part of `lyra`, and `all` makes `lyra`, shouldn't `check-mods` be removed from `all` at line 32?

Reply:
The rationale for leaving this checked in two places is that the `go version` check is very fast and fast feedback in either case (`make lyra` or `make`) is preferable.

Reply:
Cool. The check only happens once even when running "make" I think.